projects/Mallard/src/components/article/html/components/headline.ts

```diff
@@ -28,8 +29,15 @@ const getHeadline = (
             </h1>
         `
     } else {
+        const pillarInclusion =
+            pillar === 'news' || pillar === 'neutral' || pillar === 'opinion'
+        console.log(pillarInclusion)
+        const showcaseStyle =
+            articleType === ArticleType.Showcase && pillarInclusion
+                ? ` class="alt"`
+                : ``
         return html`
-            <h1>
+            <h1${showcaseStyle}>
```

Comment:
maybe put the whitespace after the h1 and remove from `showcaseStyle`? not really a big deal though

Reply:
Left it like this to keep the validity of the previous tests. Also if there is no `showcaseStyle` then we dont get an ugly gap in the rendered html